### ISSUE TITLE
ESP32 Compile error

### ISSUE THREAD

**Issue by petrkr** [bartoszbielawski/LEDMatrixDriver#12]
Library could not be compiled by using ESP32 SDK because missing `memset` function by default. It must be included separately by `#include <cstring>` Then it works fine.

patch:
```
diff --git a/src/LEDMatrixDriver.hpp b/src/LEDMatrixDriver.hpp
index b738390..26125ad 100644
--- a/src/LEDMatrixDriver.hpp
+++ b/src/LEDMatrixDriver.hpp
@@ -20,6 +20,7 @@
 #define LEDMATRIXDRIVER_H_
 
 #include <SPI.h>
+#include <cstring>
 
 class LEDMatrixDriver
 {
```

**Comment by petrkr** [comment]
Did you fixed that issue?

**Comment by bartoszbielawski** [comment]
Yes, I have just released version 0.2.0 and it includes a fix.
